data/rao-result/rao-result-json/src/test/resources/retrocompatibility/v1.5/crac-for-rao-result-v1.5.json

```diff
@@ -0,0 +1,410 @@
+{
+  "type" : "CRAC",
+  "version" : "1.5",
```

Comment:
```suggestion
  "version" : "2.4",
```

Reply:
crac-for-rao-result-v1.4.json and before also have version crac v1.5 even if it was not the last version of crac at the time (I think), do I still update the crac version?

Reply:
oh no ok sorry, I thought you created a new CRAC with the actual version. then keep 1.5

Reply:
ok